src/cte_profiler/cteProfilerService.ts

```
@@ -0,0 +1,307 @@
+  Event,
+  EventEmitter,
+  TextDocument,
+  Uri,
+  window,
+} from "vscode";
+import { CteInfo } from "../code_lens_provider/cteCodeLensProvider";
+import { DBTProjectContainer } from "../dbt_client/dbtProjectContainer";
+import {
+  CteProfileEntry,
+  CteProfileResult,
+} from "./cteProfilerTypes";
```

ESLint

[error] Replace `⏎··CteProfileEntry,⏎··CteProfileResult,⏎` with `·CteProfileEntry,·CteProfileResult·`

(prettier/prettier)